Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM alpine:edge
 
 RUN apk add --no-cache \
     # Runtime deps
-    ruby ruby-bundler ruby-bigdecimal ruby-io-console tzdata nodejs bash \
+    ruby ruby-bundler ruby-bigdecimal ruby-io-console tzdata nodejs bash git \
```

Comment:
What do we need git for?

Reply:
could not get bundler to install the git-based gems without it ...

Reply:
Got it, thanks. Please move to the line below "Bundle install deps"

Reply:
For info, list of the git based gems

```
Fetching https://github.com/rspec/rspec-rails
Fetching https://github.com/rspec/rspec-core
Fetching https://github.com/rspec/rspec-mocks
Fetching https://github.com/rspec/rspec-support
Fetching https://github.com/rspec/rspec-expectations
```

Reply:
I've removed them now (https://github.com/thredded/thredded/commit/38946f2cef65b93f7df59b3f7bdbedc292283ff2) but still let's keep `git` here as we occasionally need git gems, especially when adding support for new Rails versions.